src/averageAges.js

```diff
@@ -38,6 +56,15 @@ function calculateMenAverageAge(people, century) {
  */
 function calculateWomenAverageAge(people, withChildren) {
   // write code here
+  let women = people.filter((person) => person.sex === 'f');
+
+  if (withChildren) {
+    women = women.filter((person) =>
+      people.find((el) =>
+        el.mother === person.name));
+  }
```

Comment:
With ternary operator you can go like that:
```suggestion
  withChildren && women.filter((person) => {
    people.find((el) => {
      return el.mother === person.name;
    });
  });
```